spotbugs/src/main/java/edu/umd/cs/findbugs/sarif/Location.java

```diff
@@ -178,9 +191,8 @@ JSONObject toJSONObject() {
 
         Region(int startLine, int endLine) {
             assert startLine > 0;
-            assert endLine > 0;
             this.startLine = startLine;
-            this.endLine = endLine;
+            this.endLine = startLine == endLine ? 0 : endLine;
```

Comment:
if startline is equal endline, we should not print, since they have the same information.